pkg/v1/providers/kapp-controller-values/kapp-controller_data.lib.yaml

```diff
@@ -29,6 +29,8 @@ kappController:
     tolerations:
     - key: CriticalAddonsOnly
       operator: Exists
+    - effect: NoSchedule
```

Comment:
Question: Does management-cluster plugin use these data values to generate kapp-controller deployment yaml? cc @tenczar @anujc25

Reply:
Yes.